src/vsr/message_header.zig

```diff
@@ -721,7 +721,7 @@ pub const Header = extern struct {
             return null;
         }
 
-        pub fn reserved(cluster: u128, slot: u64) Prepare {
+        pub fn reserved_operation(cluster: u128, slot: u64) Prepare {
```

Comment:
Not a huge fan of this name, but nothing better comes to mind. Maybe `reserved_slot`? Anyway, shouldn't be a problem to rename later!